mayaUSD.mod.template

```diff
@@ -13,6 +13,7 @@ PXR_PLUGINPATH_NAME+:=lib/usd
 
 + MayaUSD_LIB ${MAYAUSD_VERSION} ${CMAKE_INSTALL_PREFIX}
 PATH+:=lib
+PATH+:=lib/gtest
```

Comment:
`lib/gtest` needs to be in the path. AL tests needs to link against gtest shared library if ran from interactive Maya.

Reply:
Technically this only needs to be added when building with the tests. Can we somehow conditionally add this based on:

if (BUILD_TESTS)

Reply:
Hey @seando-adsk that's what I tried to do but couldn't find a way to do in mod. 

We can't even add comments in mod, let alone conditional statements :)  I would love to know if there is a way to do all that.

Reply:
Well since this is a .mod.template that is run thru configure_file, couldn't you set some variable like MAYAUSD_GTEST_PATH="PATH+:=lib/gtest" when BUILDTESTS is defined and and have it empty otherwise. Then in the mod file replace line 16 by:
${MAYAUSD_GTEST_PATH}

Reply:
@seando-adsk True, good idea. I will make the change.